.github/workflows/test_helper.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           sqlfluff lint products/developments/sql/*.sql
           sqlfluff lint products/developments/sql/qaqc
           sqlfluff lint products/developments/sql/unit_change_summary --templater=jinja
+          sqlfluff lint products/facilities/facdb/sql/
           sqlfluff lint products/knownprojects/sql --templater=jinja
```

Comment:
Also might be worth starting to only lint changed files as this actually takes a good chunk of time. But again, not in this PR